### DISCUSSION TITLE
Group Assignment #2 Memo Submission

### DISCUSSION THREAD

**Discussion by albertkun** [albertkun/21S-ASIAAM-191A#92] [Assignments]
Have one person comment on this post to submit the link to your memo when you are finished.

**Comment by gu-ana** [comment]
https://docs.google.com/document/d/12e01HPhnCDphWnRJ5EHmwt0Eu-FqUv5k-ukRLZbkv2E/edit?usp=sharing

**Comment by KenNL42** [comment]
https://docs.google.com/document/d/1t5-I---plPMOixWv8uSQc-KY3Rv2N-bcpgP1tS3zeDI/edit?usp=sharing

**Comment by epham36** [comment]
Jason Muljadi, Emily Pham, Jess Xu: https://docs.google.com/document/d/1Er8Jq6M4fRsptmrNRLiWepqMFx1uWburt91TsRSp5G0/edit?usp=sharing

**Comment by lvalles10** [comment]
Group 8: Nina Long, Lauren Valles, Katie Greenler: https://docs.google.com/document/d/1b_GUOo-wERHBFgGT8oTcI8xt22tw8IuO18pYQ7WCpEQ/edit?usp=sharing

**Comment by alizezila** [comment]
Group 1: Alize Magaña, Andi Min, Anny Rodriguez Viloria  https://docs.google.com/document/d/1DH6YNUFHU2uSeaVnlBn4IzXZT_FO6lTki5XhPlSB_BA/edit?usp=sharing

**Reply by albertkun** [reply]
@alizezila I don't have access to your group's memo

**Reply by annyrviloria** [reply]
Viewable link:
https://docs.google.com/document/d/1DH6YNUFHU2uSeaVnlBn4IzXZT_FO6lTki5XhPlSB_BA/edit?usp=sharing

**Comment by Wny-Duong** [comment]
Group 4: Ericko Halim, Latisha Sumardy, Wilny Duong
https://docs.google.com/document/d/1rWZ6G-Pj3Jj16WsD7iaiicDROEX7VdmYcXNI4Tlb9cs/edit?usp=sharing

**Comment by rh5140** [comment]
Group 2: Brian Zamora @brillianz, Sam Karakas @skarakas1, Ray Hsiao @rh5140
https://docs.google.com/document/d/1GKSrchk_-NSHU-dDEOb8n-9Ya_Nxqn8qWPena-MbM0o/edit?usp=sharing
